packages/components/src/Overlay/Underlay.tsx

```diff
@@ -37,7 +37,7 @@ export const Underlay = ({
     <ModalOverlay
       className={({ isEntering, isExiting }) =>
         cn(
-          'fixed inset-0 z-40 grid place-items-center overflow-y-auto',
+          'fixed inset-0 z-40 flex flex-col items-center justify-center overflow-hidden p-4',
```

Comment:
if we remove `grid` the `grid-areas` doesn't have any effect anymore or am I wrong?